Foundation/JSONEncoder.swift

```diff
@@ -1750,6 +1750,25 @@ extension _JSONDecoder {
     fileprivate func unbox(_ value: Any, as type: Bool.Type) throws -> Bool? {
         guard !(value is NSNull) else { return nil }
 
+#if false
```

Comment:
The `#if false`es are good candidates for converting to `DEPLOYMENT_TARGET_SWIFT`